src/d3d11/d3d11_texture.cpp

```diff
@@ -408,9 +442,12 @@ namespace dxvk {
     if (pDesc->MiscFlags & (D3D11_RESOURCE_MISC_TILE_POOL | D3D11_RESOURCE_MISC_TILED))
       return E_INVALIDARG;
 
+    // Shared resources may only have 1 mip. Native clamps this.
```

Comment:
Is this really true? The documentation for [Extended Resource Sharing](https://docs.microsoft.com/en-us/windows/win32/direct3d11/direct3d-11-1-features#extended-support-for-shared-texture2d-resources) disagrees here and says that more than one mip is allowed.